src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,66 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final int AMOUNT = 1;
+    private static int supply = 0;
+    private static int buy = 0;
+    private static StringBuilder stringBuilder;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        readFromFile(fromFileName);
+        createReport();
+        writeToFile(toFileName);
+    }
+
+    private String[] readFromFile(String fromFileName) {
```

Comment:
read from file should only read data from file, than in the method createReport() you should iterate over data from file, create report, and return it